### DISCUSSION TITLE
[5 giugno 2023] Esercizio 4

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#248] [Esami]
$\Large a)$ Usando gli assiomi dell’algebra di Boole, verificare la seguente identità:

<div align="center">

![image](https://github.com/user-attachments/assets/58e4e6e1-cdd3-4d47-8c88-becfca565bb1)

</div>

$\Large b)$ Scrivere la funzione in forma POS canonica

**Comment by Dieal** [comment]
![IMG_20250108_103506_170](https://github.com/user-attachments/assets/6e856bfa-5e30-4030-8eba-9c7555985bb8)
